<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_pop_up</class>
 <widget class="QDialog" name="settings_pop_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressure unit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature unit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="temp_units">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>86</width>
     <height>30</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>°F</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>°C</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="pressure_units">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>86</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Pa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ATM</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
